Userland/Services/SystemServer/Service.cpp

```diff
@@ -35,6 +35,8 @@ ErrorOr<void> Service::setup_socket(SocketDescriptor& socket)
 {
     VERIFY(socket.fd == -1);
 
+    [[maybe_unused]] auto result = Core::System::unlink(socket.path);
```

Comment:
Why not `TRY`?

Reply:
(preferred way to ignore results is `(void)`)

Reply:
The purpose of this is to remove potential left-over of previous portal. They are not always there, and unlinking a non-existent path is considered as a failure.

Reply:
That should be a comment IMO, not totally obvious :)

Reply:
Done!